src/pages/teams/[id].tsx

```diff
@@ -14,6 +16,9 @@ const TeamPage: React.FC = () => {
   const userId = useUserId();
   const [updateUserSession] = useUpdateUserSessionMutation();
 
+  const activeUsers = useOnlineUsers();
+  const onlineUsers = activeUsers ? activeUsers.user_online.filter((u: OnlineUser) => parseInt(u.team_id) === teamId) : [];
```

Comment:
Wouldn't you want to do this filtering in the query itself?

Reply:
Yes that would be a better solution. Will look into it.

Reply:
Now I have made the filtering inside hasura instead :)